.github/workflows/release-published.yml

```diff
@@ -222,4 +222,32 @@ jobs:
         run: mvn -B clean deploy -DskipTests=true
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          TARGET_ARTIFACT_REPOSITORY: github
\ No newline at end of file
+          TARGET_ARTIFACT_REPOSITORY: github
+
+  deploy_xsd:
+      name: Upload xsds
+      needs: [ setup ]
+      runs-on: ubuntu-20.04
+      steps:
+      - name: Download liquibase xsd
+        uses: actions/checkout@v3
+        with:
+          # Relative path under $GITHUB_WORKSPACE to place the repository
+          path: liquibase-core  
+          repository: "liquibase/liquibase"
+
+      - name: Download liquibase-pro xsd
+        uses: actions/checkout@v3
+        with:
+        # Relative path under $GITHUB_WORKSPACE to place the repository
+          path: liquibase-pro
+          repository: "liquibase/liquibase-pro"
+      
+      - name: Upload to s3
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.LIQUIBASEORIGIN_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.LIQUIBASEORIGIN_SECRET_ACCESS_KEY }}
+          AWS_DEFAULT_REGION: us-east-1
+        run: |
+          aws s3 sync $GITHUB_WORKSPACE/liquibase-core/src/main/resources/www.liquibase.org/xml/ns/dbchangelog/dbchangelog-${{ needs.setup.outputs.version }}.xsd s3://liquibaseorg-origin --only-show-errors
```

Comment:
I took a look at https://github.com/liquibase/liquibase/tree/2de02c6cb0cea9fa425be05755a0643d451aa671/liquibase-core/src/main/resources/www.liquibase.org/xml/ns/dbchangelog and I could not see `xsd` files for patch versions like `4.17.x`
I am not sure how the release workflow handles patch versions, it seems `xsd` files are handled by `major.minor` so not sure what is going to happened when somebody launches this workflow for `4.19.1`

Reply:
"There won't be any `xsd`'s for patch release, if there is an `xsd` update then it automatically becomes a major release". So for this it would be still : [dbchangelog-4.19.xsd](https://github.com/liquibase/liquibase/blob/2de02c6cb0cea9fa425be05755a0643d451aa671/liquibase-core/src/main/resources/www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.19.xsd)

Reply:
👍 added the logic to handle patch releases